custom/icds_reports/views.py

```diff
@@ -1818,7 +1818,7 @@ def post(self, request, *args, **kwargs):
 
         query_month = date(year, month, 1)
         today = date.today()
-        current_month = today - relativedelta(months=1)
+        current_month = today
         if query_month > current_month:
```

Comment:
Could get rid of the vars `today` and `current_month` and make this line `if query_month > date.today()`